plugin-procfs/README.md

```diff
@@ -1,3 +1,138 @@
 # Procfs plugin
 
-Collects processes- and system-related metrics by reading the `/proc` virtual filesystem.
+Collects processes and system-related metrics by reading the [proc](<https://www.kernel.org/doc/html/latest/filesystems/proc.html>) virtual filesystem on Linux based operating systems.
+
+## Requirements
+
+- Linux operating system.
+- Read access to file system `/proc/`.
+- Read access to file system `/proc/<pid>` for process stats.
+- Access mode in [/proc/mounts](<https://docs.kernel.org/filesystems/proc.html#mount-options>) file by configuring the [hidepid](#hidepid) parameter.
+
+### hidepid
+
+This parameter is one of those that can configure Procfs permissions. If this parameter has been set with a specific value different from the default configuration, you may not be able to collect all process metrics. Here are its available configuration values :
+
+|Value|Description|
+|-----|-----------|
+|0|`Default` : Everybody may access all `/proc/<pid>/` directories|
+|1|`Noaccess` : Users may not access any `/proc/<pid>/` directories but their own|
+|2|`Invisible` : All `/proc/<pid>/` will be fully invisible to other users|
+|4|`PTraceable` : Procfs should only contain `/proc/<pid>/` directories that the caller can ptrace|
+
+## Metrics
+
+There are various information collected by this plugin relative to Kernel, CPU, memory and processes :
+
+|Name|Type|Unit|Description|Ressource|RessourceConsumer|Attributes|
+|----|----|----|-----------|---------|-----------------|----------|
+|`kernel_cpu_time`|CounterDiff|MilliSecond|Time during the CPU is busy|LocalMachine|LocalMachine|[cpu_state](#cpu_state)|
```

Comment:
Units should be written with their standard name, for example `ms` or `millisecond` for milliseconds, not `MilliSecond`.